src/xdocs/config_whitespace.xml

```diff
@@ -1446,6 +1471,32 @@ import static java.math.BigInteger.ZERO;
   &lt;property name=&quot;tokens&quot; value=&quot;DOT&quot;/&gt;
   &lt;property name=&quot;allowLineBreaks&quot; value=&quot;false&quot;/&gt;
 &lt;/module&gt;
+        </source>
+        <p>Example:</p>
+        <source>
+class Test {
+
+  public void lineBreak(String x) {
+    Integer.
+    parseInt(x); // violation, '.' is followed by whitespace
+    Integer.parseInt(x); // Ok
+  }
+
+  public void dotOperator(String s) {
+    Integer.parseInt(s); // Ok
+    Integer. parseInt(s); // violation, '.' is followed by whitespace
+  }
+
+  public void arrayDec() {
+    int[] arr = arr; // Ok
+    int [] arr = arr; // Ok
+  }
+
+  public void bitwiseNot(int a) {
+    a = ~ a; // Ok
+    a = ~a; // Ok
+  }
+}
         </source>
         <p>
          If the annotation is between the type and the array, the check will skip validation for
```

Comment:
please do me a favor and move this paragraph
to be after  this paragraph:
```
will be ignored.
        </p>
```

examples should be config + code. This note paragraph is misplaced.

Reply:
> please do me a favor and move this paragraph to be after this paragraph:
> 
> ```
> will be ignored.
>         </p>
> ```
> 
> examples should be config + code. This note paragraph is misplaced.

Sorry to say this, but I didn't get what you asked me to do could you please elaborate?..

Reply:
<p>
         If the annotation is between the type and the array, the check will skip validation for

Should I put the above entire paragraph after this?

<a href="apidocs/com/puppycrawl/tools/checkstyle/api/TokenTypes.html#INDEX_OP">
                INDEX_OP</a>
          will be ignored.
        </p>

Reply:
done